DigitalLearningSolutions.Web/Controllers/Register/RegisterDelegateByCentreController.cs

```diff
@@ -279,12 +282,12 @@ private void SetCentreDelegateRegistrationData(int centreId)
 
         private void ValidateEmailAddress(RegisterDelegatePersonalInformationViewModel model)
         {
-            RegistrationEmailValidator.ValidateCentreEmailIfNecessary(
+            RegistrationEmailValidator.ValidateEmailNotHeldAtCentreIfEmailNotYetVerified(
                 model.CentreSpecificEmail,
-                model.Centre,
+                model.Centre!.Value,

```

Comment:
Yeah, in the Register-by-centre journey it's set based on the user's claim in Index() - it's just nullable because the model extends RegistrationData which houses `Centre` and is used in other registration journeys where it's actually nullable.